# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xjdic23/errno.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./xjdclient.c.old	2003-03-30 20:04:13.000000000 +0100
+++ ./xjdclient.c	2003-03-30 20:05:12.000000000 +0100
@@ -36,6 +36,7 @@
 #include <stdlib.h>
 #include <unistd.h>
 #include <signal.h>
+#include <errno.h>
 #include "xjdic.h"
 
 #define CVERBOSE 0
--- ./xjdserver.c.old	2003-03-30 20:04:22.000000000 +0100
+++ ./xjdserver.c	2003-03-30 20:05:31.000000000 +0100
@@ -28,6 +28,7 @@
 #include <netdb.h>
 #include <stdlib.h>
 #include <signal.h>
+#include <errno.h>
 #include "xjdic.h"
 
 #define SVERBOSE 0
